front/app/containers/PasswordReset/PasswordResetSuccess.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { screen, render, fireEvent } from 'utils/testUtils/rtl';
+
+import { PasswordResetSuccess } from './PasswordResetSuccess';
+
+const mockSignInModal = jest.fn();
+
+jest.mock('events/openSignUpInModal', () => ({
+  openSignUpInModal: () => mockSignInModal(),
+}));
+
+describe('PasswordResetSuccess', () => {
+  it('renders a success message with a login button', () => {
+    render(<PasswordResetSuccess />);
+
+    expect(
+      screen.getByText('Your password has been successfully updated.')
+    ).toBeInTheDocument();
+    expect(screen.getByRole('button')).toHaveTextContent('Log in');
+  });
+
+  it('opens sign in modal after clicking on the login button', async () => {
+    render(<PasswordResetSuccess />);
+
+    const loginButton = screen.getByRole('button');
+    fireEvent.click(loginButton);
+
+    expect(mockSignInModal).toHaveBeenCalledTimes(1);
```

Comment:
I think an even better check would be to verify the log in modal is actually there, rather than whether a function has been called.

Reply:
The idea of it being a unit test is that we are limiting the scope to this "unit". Testing that the modal is shown is not the responsibility of this file. A test like that would cause a failure in this unit test if the modal work breaks. The issue with that is that it makes it hard to track an issue because you can have multiple failures in files that are typically working well. That makes it hard to track where the failure is. We can however change the wording of the test case to reflect that.

Reply:
Right, agreed 👍